[IMP] Helpdesk: Update SLAs (#11691)
odoo/documentation: content/applications/services/helpdesk/overview/sla.rst

Service level agreements (SLA)
==============================

.. |SLAs| replace:: :abbr:`SLAs (Service Level Agreements)`
.. |SLA| replace:: :abbr:`SLA (Service Level Agreements)`

A *service level agreement* (SLA) defines the level of support a customer can expect from a service
provider. |SLAs| provide a timeline that tells customers when they can expect results and keeps the
support team on target.

.. note::
   The *SLA Policies* feature is enabled by default on newly created **Helpdesk** teams.

   To turn off the feature, or edit the working hours, navigate to :menuselection:`Helpdesk app -->
   Configuration --> Helpdesk Teams`. Click on a team to open that team's configuration page.

   From here, scroll to the :guilabel:`Performance` section. To turn off the |SLAs| feature for the
   team, clear the :guilabel:`SLA Policies` checkbox.

   .. image:: sla/sla-enable.png

>       :align: center

Reviewer: Please remove the center alignment from all images :)
```suggestion
```

  Kanban card or on the ticket itself. The |SLA| is **only** applied after the priority level has
  been updated on the ticket to match the |SLA| criteria. If no selection is made in this field,
  this policy only applies to tickets marked as `Low Priority`, meaning those with zero
  :icon:`fa-star-o` :guilabel:`(star)` icons.
- :guilabel:`Tags`: Tags are used to indicate what the ticket is about. Multiple tags can be applied
  to a single ticket.
- :guilabel:`Customers`: Individual contacts or companies may be selected in this field.
- :guilabel:`Services`: This field is available only if a team has the **Timesheets** app enabled.
  This allows the ticket to link directly to a specific line on a sales order, which must be
  indicated on the ticket in the :guilabel:`Sales Order Items` field.

.. example::
   A support team needs to address urgent issues for VIP customers within one business day.

   The new policy, titled `8 Hours to close`, is assigned to the `VIP Support` team. It **only**
   applies to tickets that are assigned three :icon:`fa-star-o` :guilabel:`(star)` icons, which

>    equates to an `Urgent` priority level.

Reviewer 2: I think we could make it even clearer by speaking more directly to the actions.

For example, instead of saying the SLA applies to urgent tickets, how about rewording it to:

> The VIP Support team exclusively handles urgent tickets, as determined by the [Helpdesk configuration](link to doc section). The SLA policy, titled 8 Hours to Close, ensures these tickets are resolved within eight hours of being assigned to the team.

This phrasing ties the SLA more directly to the team's responsibilities. What do you think? :blush: 
P.S. Also, is what I'm saying accurate? :flushed: Just making assumptions based on what I'm reading!

> - :guilabel:`Helpdesk Team`: a policy can only be applied to one team. *This field is required.*

Reviewer 3: ```suggestion
- :guilabel:`Helpdesk Team`: A policy can only be applied to one team. *This field is required.*
```

> - :guilabel:`Priority`: the priority level for a ticket is identified by selecting one, two, or

Reviewer 3: ```suggestion
- :guilabel:`Priority`: The priority level for a ticket is identified by selecting one, two, or
```

  three of the :icon:`fa-star-o` :guilabel:`(star)` icons, representing the priority level on the
  Kanban card or on the ticket itself. The |SLA| is **only** applied after the priority level has
  been updated on the ticket to match the |SLA| criteria. If no selection is made in this field,
  this policy only applies to tickets marked as `Low Priority`, meaning those with zero
  :icon:`fa-star-o` :guilabel:`(star)` icons.

> - :guilabel:`Tags`: tags are used to indicate what the ticket is about. Multiple tags can be applied

Reviewer 3: ```suggestion
- :guilabel:`Tags`: Tags are used to indicate what the ticket is about. Multiple tags can be applied
```

  to a single ticket.

> - :guilabel:`Customers`: individual contacts or companies may be selected in this field.

Reviewer 3: ```suggestion
- :guilabel:`Customers`: Individual contacts or companies may be selected in this field.
```

------------------------------------

A *target* is the stage a ticket needs to reach, and the time allotted to reach that stage, in order
to satisfy the |SLA| policy. Any stage assigned to a team may be selected for the :guilabel:`Reach
Stage` field.

Time spent in stages selected in the :guilabel:`Excluding Stages` field are **not** included in the
calculation of the |SLA| deadline.

.. example::

>    An |SLA| titled `8 Hours to Close` tracks the working time before

Reviewer 3: Not sure why, but |SLA| builds out to "SLAs" (plural) in the runbot. Can we revise the abbreviation and how it builds so it can have singular and plural? 

![image](https://github.com/user-attachments/assets/44370e8d-396f-4638-9326-e5595fcbc287)

   a ticket is completed, and would have `Solved` as the :guilabel:`Reach Stage`. Simultaneously, an

>    |SLA| titled `2 Days to Start` tracks the working time before

Reviewer 3: Same comment as above, shows up as plural in runbot

   work on a ticket has begun, and would have `In Progress` as the :guilabel:`Reach Stage`.

Meet SLA deadlines
==================

> As soon as it is determined that a ticket fits the criteria of an |SLA| policy, a deadline is

Reviewer 3: Same comment as above, shows up as plural in runbot

> calculated. The deadline is based on the creation date of the ticket, and the targeted working

Reviewer 3: ```suggestion
calculated. The deadline is based on the creation date of the ticket, as well as the targeted working
```

hours.

.. note::

>    The value indicated next to the :guilabel:`Working Hours` field of an |SLA| policy is used to

Reviewer 3: Same comment as above, shows up as plural in runbot

   determine the deadline. By default, this is determined by the value set in the :guilabel:`Company
   Working Hours` field under :menuselection:`Settings app --> Employees --> Work Organization`.

The deadline is then added to the ticket, as well as a tag indicating the name of the |SLA| applied.

.. image:: sla/sla-open-deadline.png
   :align: center
   :alt: View of a ticket's form emphasizing an open SLA deadline on a ticket in Odoo Helpdesk.

> When a ticket satisfies an |SLA| policy, the |SLA| tag turns green, and the deadline disappears from

Reviewer 3: Same comment as above, shows up as plural in runbot

view on the ticket.

.. image:: sla/sla-deadline.png
   :align: center
   :alt: View of a ticket's form emphasizing a satisfied SLA in Odoo Helpdesk.

.. important::

>    If a ticket fits the criteria for more than one |SLA| the earliest occurring deadline is

Reviewer 3: Same comment as above, shows up as plural in runbot

>    If a ticket fits the criteria for more than one |SLA| the earliest occurring deadline is

Reviewer 3: ```suggestion
   If a ticket fits the criteria for more than one |SLA|, the earliest occurring deadline is
```

Analyze SLA performance
=======================

The :guilabel:`SLA Status Analysis` report tracks how quickly an |SLA| is fulfilled, as well as the
performance of individual team members. Navigate to the report, and corresponding pivot table, by
going to :menuselection:`Helpdesk app --> Reporting --> SLA Status Analysis`.

Pivot view
----------

> By default, the report displays in a :guilabel:`Pivot` view. Any |SLA| policies in the database with

Reviewer 3: Same comment as above, shows up as plural in runbot

Graph view
----------

Switch to the graph view by selecting the :icon:`fa-area-chart` :guilabel:`(graph view)` icon at the

> top-right of the screen. To switch between the different charts, select the *related icon* at the

Reviewer 3: I've received feedback to try to avoid screen-specific directions in tutorials and documentations as they might appear differently depending on the screen, so I'd suggest being more general about icon location

          A pie chart compares data among a small number of categories.

.. tip::

>    Both the *bar chart* and *line chart* can utilize the *stacked* view option. This presents two

Reviewer 3: Should these be bolded or under gui label?

Cohort view
-----------

The *cohort* view is used to track the changes in data over a period of time. To display the
:guilabel:`SLA Status Analysis` report in a cohort view, click the :icon:`oi-view-cohort`

> :guilabel:`(cohort)` icon, in the top-right corner, next to the other view options.

Reviewer 3: As mentioned in earlier comment, I'd suggest staying general about locations on the screen
